madlibs.js

```diff
@@ -22,21 +22,110 @@
  *  { word: "store", pos: "noun" }
  *  { word: "," }
  *  ....
- * 
+ *
  * There are multiple ways to do this, but you may want to use regular expressions.
  * Please go through this lesson: https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures/regular-expressions/
  */
+
 function parseStory(rawStory) {
-  // Your code here.
-  return {}; // This line is currently wrong :)
+  let arr = rawStory.split(" ");
```

Comment:
```suggestion
  const arr = rawStory.split(" ");
```

Refrain from using `let` and use `const` instead. Unless you want to change the value of this variable later on.